### DISCUSSION TITLE
Show Table Of Contents on the right side to avoid confusion

### DISCUSSION THREAD

**Discussion by tillsanders** [outline/outline#3214] [Ideas]
Hi there – just a small idea:

I really like the little TOC feature! However, I often find myself confusing it with the actual navigation. Especially on overview sites where there are naturally many parallels between my headings and my sub-pages. I'm not sure if this can be entirely avoided, but I think it might help if the TOC would be moved to the right side of the content?

![Bildschirmfoto 2022-03-08 um 17 52 08](https://user-images.githubusercontent.com/1533846/157286817-8f31b56c-b3b8-4b10-bead-d6f754895452.png)

Have a nice day!
Till

**Comment by tommoor** [comment]
The right will have comments soon so it's not an easy shift to just flip it over, we'll keep this feedback in mind though

**Comment by EduNetEurope** [comment]
There are two duplicate requests. Can they be merged?

- https://github.com/outline/outline/discussions/6035
- https://github.com/outline/outline/discussions/6037

**Comment by ReessKennedy** [comment]
There's actually a simple way to do this. ✅ 🎉 ... not sure why I didn't figure this out sooner. Wrote up solution at link below but's basically just a CSS mod with optional JS addition to make the table of contents show up by default: 

[https://www.reesskennedy.com/right-sidebar-outline-wiki/](https://www.reesskennedy.com/right-sidebar-outline-wiki/)

![BeforeAndAft](https://github.com/outline/outline/assets/3970267/4a92b4bc-0db5-4d2b-a3c4-b52ebbc3aaa3)

**Comment by EduNetEurope** [comment]
@tommoor 
I think you can close this request. :-)

**Comment by tommoor** [comment]
This feature was added as a workspace preference in [v0.78.0 (release)](https://github.com/outline/outline/releases/tag/v0.78.0)
